Library/Formula/tachyon.rb

```diff
@@ -0,0 +1,25 @@
+class Tachyon < Formula
+  homepage "http://tachyon-project.org/"
+  url "https://github.com/amplab/tachyon/releases/download/v0.6.0/tachyon-0.6.0-bin.tar.gz"
+  sha1 "74c17c9fb186740246cc23953c8811c47bdf699e"
```

Comment:
Our new policy is to use sha256 for new objects; can you update the checksum, please?